web/src/page_params.ts

```diff
@@ -22,6 +22,12 @@ export const page_params: {
         order: number;
         type: number;
     }[];
+    custom_profile_field_types: {
+        PRONOUNS: {
+            id: number;
+            name: string;
+        }
+    };
```

Comment:
You should probably add the full set from https://zulip.com/api/register-queue while you're at it.

Reply:
I'm a bit confused by the documentation, do all of the field types have this shape?

```
{
    id: number;
    name: string;
}
```

Reply:
I'm going to assume so and continue with that assumption.

Reply:
Yep. The section on the list of things that will be present here is a bit long, and maybe misplaced (We could probably move that list to the `name` field, right?). Started https://chat.zulip.org/#narrow/stream/412-api-documentation/topic/custom_profile_field_types/near/1725241 to discuss.

![image](https://github.com/zulip/zulip/assets/2746074/ae25267a-dccc-4ec8-a43b-b0d10d8be769)

But as you can see at the bottom, it's always a 2-field data structure.